RUFAS/biophysical/animal/ration/ration_optimizer.py

```diff
@@ -149,15 +161,31 @@ def set_constraints(self, arguments: tuple[RationConfig]) -> None:
             self.fat_constraint,
             self.DMI_constraint_upper,
             self.DMI_constraint_lower,
+            self.NASEM_net_energy_constraint
         ]
 
-        self.cow_constraints = [{"type": "ineq", "fun": func, "args": arguments} for func in self.constraint_functions]
-
-        self.heifer_constraints = [
-            cons
-            for cons in self.cow_constraints
-            if cons["fun"] not in [self.NE_total_constraint, self.NE_lactation_constraint]
-        ]
+        if arguments[0].nutrient_standard is NutrientStandard.NRC:
+            self.cow_constraints = [
+                {"type": "ineq", "fun": func, "args": arguments}
```

Comment:
```suggestion
                {"type": "ineq", "function": func, "args": arguments}
```
"fun" feels way too abbreviated.

Reply:
Unfortunately this is the precise format needed for `minimize` from `scipy` to work properly, so we're a bit stuck with it for the moment!